index.html

```diff
@@ -5,65 +5,72 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=#, initial-scale=1.0">
-    <title>Document</title>
+    <title>Danilo's Portfolio</title>
 </head>
 
 <body>
     <section>
-        <header>Hello, this is my first header</header>
-        <img src="1611913973379.jpg" width="500" height="600">
+        <body>
```

Comment:
In line 11 you already declared the opening tag for the `body`. This should be removed.